content/help/addons/rich-editor.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Rich Editor (Beta)
+tags:
+    - Features
+    - Addons
+    - Rich Editor
+    - Advanced-editor
+    - editor
+category: help
+menu:
+    help:
+        parent: addons
+versioning:
+    added: 2.7
+---
+
+## Overview
+
+The Rich Editor adds intuitive styling and formatting to a posts and documents on Vanilla.
+
+![A full view of editor](/img/help/addons/rich-editor/editor-view.png)
+
+_Please note that Rich Editor is currently in beta and **is under active development**. Please report any bugs you encounter to your CSMs or on the [Vanilla Issue Tracker](https://github.com/vanilla/vanilla/issues)._
+
+## Important Functions
+
+Once enabled, Rich Editor provides your users with familiar formatting tools. Here is a quick rundown of the functions that are currently available:
+
+### Text Formatting
+
+![The bold, italic, strikethrough, code, and link buttons](/img/help/addons/rich-editor/inline-formats.png)
+
+This first set of buttons allows you to add bold, italic, strike, code, or link formatting to selected text. This menu is dynamic and contextual, appearing wherever the user makes a selection. The standard keyboard shortcuts, `ctrl+b`, `ctrl+i`, and `ctrl+k` also work when you have begun typing inside the editor.
```

Comment:
I would explicitly say what the shortcut accomplishes, even though it might be obvious to novice to advanced users.